### ISSUE TITLE
How is org-mode initialized?

### ISSUE THREAD

**Issue by borgauf** [daedreth/UncleDavesEmacs#14]
I've looked through your init.el and config.org and I can't seem to find where you specifically load the latest org-mode (org, org-extra-contrib) packages. Or are you not loading the latest org-mode repo?

**Comment by daedreth** [comment]
org-mode is initialized by default, I do not load any packages other than what you see at the bottom of config.org.
Do you believe I should be grabbing a newer version of org-mode from somewhere? If so, please point me to the source as I'd love to get that!

**Comment by borgauf** [comment]
Yes, I believe you do want to have a way to get latest-greatest org-mode -- and no, I don't believe you were doing that. Have a look at [this](https://github.com/stembasespace/sbs.emacs.d) and go down to the Org-mode header. It begins like this:

```
(use-package org
  :pin org
  :ensure org-plus-contrib
  :bind (("C-c a" . org-agenda)
	 ("C-c c" . org-capture)
	 ("C-c l" . org-store-link))
  :config
  (setq org-ellipsis " ")
  ;; Fontify org-mode code blocks
  (setq org-src-fontify-natively t)
...

```

This in effect calls the latest-greatest org-plus-contrib which contains the latest-greatest org.el. Also, my init.el makes certain if anything isn't there on first install, it will go out and get it. Still, I think you need to periodically do a list-packages to get updates. So as you can see this is a (zum heulen) hack of your fine effort.
